lib/phoenix_live_view/route.ex

```diff
@@ -68,7 +69,7 @@ defmodule Phoenix.LiveView.Route do
 
     case Phoenix.Router.route_info(router, "GET", route_path, host) do
       %{plug: Phoenix.LiveView.Plug, phoenix_live_view: lv, path_params: path_params} ->
-        {view, action, opts, {live_session_name, _live_session_opts, vsn}} = lv
+        {view, action, opts, {live_session_name, live_session_extra, vsn}} = lv
```

Comment:
Perhaps we should make the third element a map altogether?

This way we can simplify a couple lines below too and store only `live_session` instead of live_session_name, live_session_vsn, live_session_extra, etc.

Reply:
Ready for another look! :)